packages/amplify-category-function/src/provider-utils/awscloudformation/utils/consolidateDependsOn.ts

```diff
@@ -0,0 +1,30 @@
+import { $TSMeta } from 'amplify-cli-core';
+import { FunctionDependency } from 'amplify-function-plugin-interface';
+import _ from 'lodash';
+import { category } from '../../../constants';
+import { lambdaLayerSetting, resourceAccessSetting, ServiceName } from './constants';
+
+export const consolidateDependsOnForLambda = (
+  projectMeta: $TSMeta,
+  currentDependsOn: FunctionDependency[],
+  lambdaToUpdate: string,
+  selectedSettings: string[],
+): FunctionDependency[] => {
+  let updatedDependsOn: FunctionDependency[];
+  const prevFunctionParametersDependsOn: FunctionDependency[] = _.get(projectMeta, ['function', lambdaToUpdate, 'dependsOn'], []);
+  if (selectedSettings.includes(resourceAccessSetting)) {
+    // insert old lambdaLayer dependsOn if present
+    const prevLayersDependsOn: FunctionDependency[] = prevFunctionParametersDependsOn.filter(
+      resource => _.get(projectMeta, [category, resource.resourceName, 'service'], []) === ServiceName.LambdaLayer,
```

Comment:
It looks like the default value in `_.get()` (here and on line 25) is not needed.